src/lib/constants.js

```diff
@@ -1,4 +1,5 @@
 export const COLUMN_COUNT_QUERY_KEY = 'columns';
-export const LOCAL_STORAGE_NAMESPACE = 'music-markdown';
-export const REPO_REGEX = '/repos/:repo([^/]+/[^/]+)';
+export const WINDOW_STORAGE_NAMESPACE = 'music-markdown';
```

Comment:
Renamed this to allow future storage for `sessionStorage`, such that we are not blindly utilizing `localStorage` unnecessarily.